core/include/gnuradio-4.0/Port.hpp

```diff
@@ -268,6 +268,11 @@ concept ConsumablePortSpan = ConsumableSpan<T> && requires(T span) {
     { *span.tags.begin() } -> std::same_as<const gr::Tag&>;
 };
 
+template<typename T>
+concept PublishablePortSpan = PublishableSpan<T> && requires(T span, property_map& tagData, Tag::signed_index_type index) {
```

Comment:
Merge with PublishableSpan? Are there use-cases where we need to keep them separate?

Reply:
@wirew0rm the question is about
 a) naming/proliferation of concepts (similar dependent definitions)
 b) we need this named concept since users -- with this PR -- do not usually interact directly with the buffer-related concept.